core/utils.py

```diff
@@ -0,0 +1,35 @@
+import jwt
+import json
+import requests
+
+from django.http import JsonResponse
+from django.core.exceptions import ObjectDoesNotExist
+
+from my_settings import SECRET_KEY
+from user.models import User
+
+class Login_required:
```

Comment:
class 명은 첫글자는 대문자로 해서 CamelCase 로 작성을 부탁 드립니다.
```suggestion
class LoginRequired:
```